pkg/controller/worker/actuator.go

```diff
@@ -72,13 +80,19 @@ func (d *delegateFactory) WorkerDelegate(ctx context.Context, worker *extensions
 		return nil, err
 	}
 
+	dvManager, err := kubevirt.NewDefaultDataVolumeManager(kubevirt.ClientFactoryFunc(kubevirt.GetClient))
```

Comment:
Please inject this via the delegate factory. This way you could use the same instance that's injected in the actuator.

Reply:
Yeah, good point.